WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -27,7 +27,7 @@ public TorProcessManager(TorSettings settings, EndPoint torSocks5EndPoint)
 			TorSocks5EndPoint = torSocks5EndPoint;
 			TorProcess = null;
 			Settings = settings;
-			TorSocks5Client = new TorSocks5Client(torSocks5EndPoint);
+			TorSocks5ClientFactory = new TorSocks5ClientFactory(torSocks5EndPoint);
```

Comment:
I don't think we want to use the pool here. But `TorSocks5ClientFactory.IsTorRunningAsync` can be moved to `TorProcessMonitor` because it's the only place we really use it.